agent/engine/execcmd/manager_init_task_linux.go

```diff
@@ -77,111 +77,96 @@ var (
 		"SessionWorkersLimit": %d
 	  }
 	}`
-	execAgentConfigFileNameTemplate = `amazon-ssm-agent-%s.json`
+	execAgentConfigFileNameTemplate    = `amazon-ssm-agent-%s.json`
```

Comment:
we'll still need to add the seelog.xml template here.  (I think that's a separate task if so, we proabaly need a TODO for that so it doesn't get lost.)

Reply:
Will, add TODO.